packages/create-playkit-plugin/createPlaykitPlugin.js

```diff
@@ -129,19 +133,74 @@ if (program.info) {
         .then(console.log);
 }
 
-if (typeof projectName === 'undefined') {
-    console.error('Please specify the project directory:');
-    console.log(
-        `  ${chalk.cyan(program.name())} ${chalk.green('<project-directory>')}`
-    );
-    console.log();
-    console.log('For example:');
-    console.log(`  ${chalk.cyan(program.name())} ${chalk.green('my-react-app')}`);
-    console.log();
-    console.log(
-        `Run ${chalk.cyan(`${program.name()} --help`)} to see all options.`
-    );
-    process.exit(1);
+async function askProjectOptions() {
```

Comment:
general comment
pressing `ctrl+c` to exit the process while asking the questions just chose the default value instead, it doesn't respect the signal to exit the program

![image](https://user-images.githubusercontent.com/4751797/69943916-adddfd00-14ee-11ea-92c2-d16c95a92194.png)